content/communicators-groups.rst

```diff
@@ -10,25 +10,68 @@ Communicators and groups
 
 .. objectives::
 
-   - Learn about tagging of messages and why it is an unsafe mechanism to distinguish library and host messages.
-   - Learn the difference between :term:`intracommunicator` and :term:`intercommunicator`.
-   - Learn how to create communicators by splitting with |term-MPI_Comm_split|.
+   - Learn about tagging of messages and why it is an unsafe mechanism to
+     distinguish library and host messages.
+   - Learn the difference between :term:`intracommunicator` and
+     :term:`intercommunicator`.
+   - Learn how to create new communicators.
 
 
+.. typealong:: Something with intercommunicators
 
-.. todo::
+   sdfasdfoaj
```

Comment:
is this intentional?

Reply:
yes, just placeholder text to avoid a warning from Sphinx :smile_cat: